library/pk.c

```diff
@@ -378,6 +378,216 @@ int mbedtls_pk_can_do_ext(const mbedtls_pk_context *ctx, psa_algorithm_t alg,
 }
 #endif /* MBEDTLS_USE_PSA_CRYPTO */
 
+#if defined(MBEDTLS_PSA_CRYPTO_CLIENT)
```

Comment:
I'm getting confused: should `mbedtls_pk_get_psa_attributes` exist whenever `MBEDTLS_PSA_CRYPTO_CLIENT` exists, or only when `MBEDTLS_PSA_CRYPTO_C` exists? I thought we wanted to try to make the pk module work with `MBEDTLS_PSA_CRYPTO_CLIENT`. But `component_test_psa_crypto_client` fails as of 64996c3be901c62636cfa2d88e92f2c31118bef5 because `mbedtls_ecc_group_to_psa` is not available.